app/services/katello/pulp3/content_view_version/importable_repositories.rb

```diff
@@ -2,13 +2,18 @@ module Katello
   module Pulp3
     module ContentViewVersion
       class ImportableRepositories
-        attr_accessor :creatable, :updatable, :organization, :metadata
+        attr_accessor :creatable, :updatable, :organization, :metadata, :redhat
 
-        def initialize(organization:, metadata:)
+        def initialize(organization:, metadata:, redhat: false)
           self.organization = organization
           self.metadata = metadata
           self.creatable = []
           self.updatable = []
+          self.redhat = redhat
```

Comment:
Was there any consideration given to making this class deal with Custom repos and creating a separate class for Red Hat repos? I think evaluating the code duplication against the clarify of having two separate classes (less if statements, branches thru code) is worth it here, even if the answer is "No this is fine the way it is"

Reply:
We had that originally, but Partha suggested we go with what we have now, we have a separate class for the RH repo creation.

Reply:
@jturel He did have separate classes. But I noticed duplicate logic. If  you prefer them being in separate classes I recommend putting the duplicate logic in a mixin and do an include.

Reply:
About 90% of the code was same